dandi/support/digests.py

```diff
@@ -137,3 +137,42 @@ def md5file_nocache(filepath: str | Path) -> str:
     present in Zarrs
     """
     return Digester(["md5"])(filepath)["md5"]
+
+
+DIGEST_PRIORITY = ("dandi-etag", "zarr-checksum", "sha512", "sha256", "sha1", "md5")
+"""
+Default order to check file digests against.
+"""
```

Comment:
```suggestion
#: Default order to prioritize digest types for `check_digests()`
DIGEST_PRIORITY = ("dandi-etag", "sha512", "sha256", "sha1", "md5")
```

Reply:
I removed the `zarr-checksum` entry because that does not apply when checksumming a single file.

Reply:
Got it. Curious why make this a regular comment vs. a docstring that could get picked up by sphinx? seems useful to annotate what this is, but y'all seem to do docs different than me so happy to do whatever :)

Reply:
[Comments that start with `#:` will be picked by Sphinx](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autofunction), and that's currently the style this codebase uses.

Reply:
aha! ok i have never seen that style. will change